src/webviews/mixins/reporter.ts

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Vue from 'vue'
+import { WebviewClientFactory, GlobalProtocol } from '../client'
+
+/**
+ * Mixin for instrumenting components for testing.
+ */
+const reporter: Vue.ComponentOptionsMixin = {
```

Comment:
Nit: rename to `testReporter` or something. Make sure the user knows its for testing.